observation_portal/proposals/models.py

```diff
@@ -227,6 +228,15 @@ def as_dict(self, exclude=None):
         time_allocation = model_to_dict(self, exclude=exclude)
         return time_allocation
 
+    def save(self, *args, **kwargs):
+        tas_count = TimeAllocation.objects.filter(proposal=self.proposal, semester=self.semester,
+                                                  instrument_types__overlap=self.instrument_types).count()
+        if tas_count > 1:
```

Comment:
now I'm not sure... I need to write some more unit tests for this I think. I tested the form submission well but then this condition is never called, since this should only apply if you get past the form validation. I'll add some more unit tests that directly save the TimeAllocations to check if this condition is right